site/docs/build-ref.html

```diff
@@ -357,6 +357,29 @@ <h4>Package names, <code>//<b>package-name</b>:...</code></h4>
   end with a slash.
 </p>
 
+<h4 id="pattern_expansion">Target pattern expansions, <code>//<b>...</b></code>
+  and <code>//<b>:all</b></code></h4>
+<p>
+  The two <code>...</code>and <code>:all</code> notations are special when used
+  on the command line. These target patterns are expanded by bazel before the
+  build is started.
+</p>
+<p>
+  Adding <code>...</code> after a package tells bazel that you're interested in
```

Comment:
Nit here and below:  I think we capitalize "Bazel" when we're not referring to the command line, which I think fits this context.